storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -809,32 +809,31 @@ func newPersistentVolume(
 		)
```

Comment:
minor: `s/templace/template` in commit message

Reply:
Done